include/predict/predict.h.in

```diff
@@ -370,6 +370,17 @@ predict_julian_date_t predict_next_los(const predict_observer_t *observer, predi
  **/
 double predict_doppler_shift(const predict_observer_t *observer, const predict_orbit_t *orbit, double downlink_frequency);
 
+/**
+ * Calculate squint angle for satellite, i.e. the offset of the satellite antenna with respect to the QTH antenna. Is defined only for SDP4 satellites for now.
```

Comment:
I'd prefer if the last sentence was moved to a \remark or \note. It will stand out more clearly in the API doc then.

There's also a small typo in that sentence: "Is" should be "It's".